tests/unit/kvdb/kvdb_home_test.c

```diff
@@ -5,6 +5,7 @@
 
 #include <limits.h>
 #include <string.h>
+#include <sys/stat.h>
```

Comment:
Curious, why so many seemingly random changes to the list of include files?

Reply:
Somewhere along the way someone was requiring hse_util/base.h when it didn't need to, and this breaks all the consumers of that header file.

Reply:
Specifically I think hse_util/hse_err.h was requiring it, but I removed the include and used standard C11 integers.